app/controllers/participant_signup_controller.rb

```diff
@@ -0,0 +1,33 @@
+class ParticipantSignupController < ApplicationController
+	before_filter :fetch_participant
+  	skip_before_filter :check_authorization	
+
+
+	def index
+	  redirect_to :action => :participant_signup_intake_form
+	end
+
+	def intake_form
+	  unless current_user && current_user.person.can_view?(@participant)
+        return render_error("You are not allowed to view this form")
+      end
+	  if request.put?
+     	@participant.intake_form_signature = params[:participant][:intake_form_signature]
```

Comment:
The Participant's `intake_form_signature` should be updated in the call to `update_attributes` below. Is there any reason for the first `save` that's separate from the second `update_attributes`?